engines/agi/text.cpp

```diff
@@ -889,9 +928,21 @@ void TextMgr::stringEdit(int16 stringMaxLen) {
 
 	// Caller can set the input string
 	_inputStringCursorPos = 0;
-	while (_inputStringCursorPos < inputStringLen) {
-		displayCharacter(_inputString[_inputStringCursorPos]);
-		_inputStringCursorPos++;
+	if (!_vm->isLanguageRTL()) {
+		while (_inputStringCursorPos < inputStringLen) {
+			displayCharacter(_inputString[_inputStringCursorPos]);
+			_inputStringCursorPos++;
+		}
+	} else {
+		while (_inputStringCursorPos < inputStringLen) 
+			_inputStringCursorPos++;
+		if (stringMaxLen == 30)
+			// called from askForSaveGameDescription
+			charPos_Set(_textPos.row, 34 - _inputStringCursorPos);
```

Comment:
More magic numbers

Reply:
OK.

Reply:
Indeed.
If you have some not magic suggestion that makes sense, I'll be happy...